### ISSUE TITLE
Negative Electrondensities

### ISSUE THREAD

**Issue by mesalas** [tdgrant1/denss#4]
A main assumption for the regularization procedure in DNESS is that the excess electron density is positive. This is a very reasonable assumption when considering proteins and has proved to be very effective. However, for other nano sized particles, such as surfactant micelles for example, this is not the case.

Im wondering if you have considered what would happen if you relaxed the positive excess  electron density criteria? Im well aware that such a relaxation may just result in being unable to arrive at a consensus model, as the degrees of freedom are simply too great.

In my experience the presence of negative and positive excess electron densities, can give rise to some quite distinct features in the scattering pattern that, to some degree, could constrain the possible solution space. 

Additionally, it may also be possible to give an estimate of the factions of the positive and negative electron densities and to compare with scattering data measured on absolute scale which along with a known particle concentration imposes additional constraints for the regularization. 

In short, Im curious what would happen if you allow for negative electron densities? Given additional features in the experimental data and/or additional regularization parameters (absolute scale and particle concentration), it may make one able to apply the DENSS procedure to scattering data origination form particles other than proteins.

**Comment by tdgrant1** [comment]
Hi Nicholas,

Yes you are correct and DENSS can actually do that, and in fact there is
now an option (for the last month or so maybe) to make this more
convenient. You can invoke the " -m MEMBRANE " option, which will allow for
negative electron densities and use only shrink-wrap as the primary
restraint.  So while it's called the MEMBRANE mode, it's not limited to
just membrane proteins but works for anything really, just does exactly
what you're saying, to turn off the positivity restraint.  In my tests it
does a pretty decent job of getting the right shape, though the actual
density values aren't reliable in some cases.

Does that answer your question?

Best,
Tom

--
Thomas D. Grant, Ph.D.
Research Assistant Professor
Department of Structural Biology
BioXFEL
University at Buffalo
Hauptman-Woodward Medical Research Institute
Rm 209, 700 Ellicott St.
Buffalo, NY 14203

**Comment by mesalas** [comment]
Thanks for the answer. That’s pretty cool. I’ll try it out with some of my old data and let you know how it works out.
